app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,7 @@
+class CustomersController < ApplicationController 
+  def show 
+    @customers = Customer.all  
```

Comment:
For this show page, we can expect the user will type a customer's ID in the url (i.e. /customers/1) so we want to find that specific customer: `@customer = Customer.find(params[:id])`